src/core/engines/Cesium/Feature/Raster/hooks.ts

```diff
@@ -0,0 +1,169 @@
+import { VectorTileFeature } from "@mapbox/vector-tile";
+import { ImageryLayerCollection, ImageryProvider, WebMapServiceImageryProvider } from "cesium";
+import { MVTImageryProvider } from "cesium-mvt-imagery-provider";
+import { useEffect, useMemo, useRef, useState } from "react";
+import { useCesium } from "resium";
+
+import { ComputedFeature, ComputedLayer, Feature } from "@reearth/core/mantle";
+import { evalFeature } from "@reearth/core/mantle/evaluator/simple";
```

Comment:
```suggestion
```

evalFeature should be passed from props:

mantle's atom -> Engine -> Cesium -> Cesium/Feature -> Cesium/Feature/Raster

Becasue:

- Allowing unlimited imports makes the code difficult to read codes. To avoid such confusion, for example, modules should be exported that are used externally from index.ts only.

```ts
// 🤔 
import { evalFeature } from "@reearth/core/mantle/evaluator/simple";
// 👍 
import { ComputedFeature, ComputedLayer, Feature, evalFeature } from "@reearth/core/mantle";
```

- evalFeature will support multiple strategies in the future. Also, you may want to have cache functionality. Therefore, please use evalFeature via atom instead of directly using evalFeature for the simple layer.

Reply:
I fixed this comment.
[refactor: evalFeature](https://github.com/reearth/reearth-web/pull/388/commits/ab5fb92a82da1adfde9c45c7ef982a69859eb54f)